src/main/java/uk/gov/hmcts/reform/draftstore/service/validation/JsonObjectValidator.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.draftstore.service.validation;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class JsonObjectValidator implements ConstraintValidator<JsonObject, JsonNode> {
+
+    @Override
+    public void initialize(JsonObject constraintAnnotation) {
+
```

Comment:
in terms of Sonar don't we need to add a comment here?

Reply:
SonarQube is not used on this project